### DISCUSSION TITLE
SavePersianFighters.com, a website to follow up lost/arrested people

### DISCUSSION THREAD

**Discussion by zaaraa-z** [WeAreMahsaAmini/Core#478] [Ideas]
Hi,
It seems lots of people are out there - specially in small towns - arrested/kidnapped/lost/tortured, and **nobody even has heard of them!**

Like "celebritywall", we can develop a website to add the names and information plus a description of what has happened, in which city, comments fileds, potential to mention authorities, etc. ...

We can color-code pictures based on updates we get.
For now, I don't even care about the data validation as there is no way to do that (unless we promote the website so that journalists can give us updates).

Please think about this. We should be the voice of protestors' families.

Zara.

**Comment by redflake** [comment]
If you can provide a hosting, domain, and you can update the names of nobody even has heard, I'm ready to create the page, then you might ask for more support

**Reply by introvertAnonymous** [reply]
Thanks for this idea. I'm working on this idea to collect information as much as possible from everyone who went missing. [This](https://docs.google.com/spreadsheets/d/1uhfm9FWCcB7qw_i7LpJiP219CxXf_nPC8UiM_X-IQgk/edit#gid=1028452535) Google sheet is getting constantly updated. I'm using it for creating a website to aggregate all names and using Twitter API count the amount of times their name has been used as hashtags over time. I have also tried to ingest some of this data to Wikidata, [example](https://www.wikidata.org/wiki/Q115574569). However, Wikidata needs references and more description to keep this data, otherwise they will delete them.  
I will open source in the upcoming days, once it reaches a state that is publishable. Lol  
I'm preparing all the initial data and infrastructures ready, but to keep them up to date and provide references, context and descriptive information in English and Persian for ingesting them into Wikidata, I will need volunteers. My goal is to monitor the amount of support each person get and make sure no voice gets silenced.  
I'm going to need any help you can give for designing and implementing the UI app.   
This is how the simple app I have implemented so far looks like:
![profile](https://i.ibb.co/Q9955kW/Screenshot-from-2022-12-09-20-02-44.png)

**Reply by redflake** [reply]
Great Job!
let me know, when to work on your repository.

**Reply by zaaraa-z** [reply]
wonderful guys!
I will check everything tomorrow and will write right back to see how we can share tasks.

**Reply by zaaraa-z** [reply]
Seems nice. We need to make the website more informatve as well as productive.
I will help in UI design and frontend developmenet, also whenever find time, will update information.
Please give us an update asa we could have access to repo.

Are you using this Google doc as a source of injecting data to the website?
I wish you promote it somehow. It seems like a very nice document.

**Reply by introvertAnonymous** [reply]
Thanks, I will update you as soon as I can deploy the first version.  
There are some activists who are actively monitoring and updating this file, [this](https://twitter.com/followupiran) is their Twitter account.  
There are other groups like [HRANA](https://twitter.com/HRANA_English) and [SedayeShahrivar](https://twitter.com/SedayeShahrivar), and also [1500Tassvir](https://twitter.com/1500tasvir_list) that they are publishing the name of people in their Twitter page.  
My ultimate goal is to scrap this list from these accounts and then aggregate them and actively monitor each person's name in Twitter to see which ones are not getting the right amount of hashtag usage.

**Reply by introvertAnonymous** [reply]
I deployed the first version [here](http://incarcerated-iranian.com/).  
I've made [this repo](https://github.com/introvertAnonymous/Incarcerated_iranian) public, not sure how can I migrate it here. The current code base for UI is extremely inefficient though, as I have never been a UI developer. Above that, there are multiple UI components which doesn't work!  
I have a couple of ideas to make it more useful.  
Currently, for security reasons, to update the information in the database, you need to log in. I won't share the credentials here.
My next step is to integrate a Twitter scraper into the backend to show the most recent tweets based on the input hashtags. So it's easier to complete the missing information from social media.

**Reply by redflake** [reply]
Thank you @introvertAnonymous for your effort
I would like to start to work on UI and Backend
you can also create tasks or issues and assign it to me

**Reply by introvertAnonymous** [reply]
Thanks. Sorry for late response, I've had a hectic day in my today's work.  
I can handle the backend parts, but for the UI, not sure whether it makes sense to use the current code base or is it more efficient to start from scratch. I can give you all the information we can have from each person in each page, and come up with a way to ask people to add new people or complete the missing information in our application manually.   
For the start, I think, we can use the [1500tasvir](https://twitter.com/1500tasvir_list) account to add those people, but the main goal is to help less famous people who don't have voice to talk about them.

**Comment by aggie-for** [comment]
I love this! I’m a UI/UX designer.  Let me know how I can help? I’m pretty good with sketch, figma, adobe. And specialize in mobile apps.

**Reply by introvertAnonymous** [reply]
Thanks. I think there's a lot we can do. I'm struggling with some deployment issues, atm. So rn [it's](http://incarcerated-iranian.com) on http. Here are the page, I think we need to do a session of brainstorming to figure out how we can visualize this info in a better way and how to insert and improve this UI.
![dashboard](https://i.ibb.co/MV41vyP/Screenshot-from-2022-12-13-02-53-16.png)
![peopleList](https://i.ibb.co/3NTXd4L/Screenshot-from-2022-12-13-02-58-13.png)
![personDetails](https://i.ibb.co/L1WdJkn/Screenshot-from-2022-12-13-02-52-47.png)

**Comment by zaaraa-z** [comment]
Hi,
I want to add some styles. Rarely worked with React. How do you prefer to deal with styles? If you can, just add css files in related components or a general one. Then I will modify styles in those files.

**Reply by introvertAnonymous** [reply]
Unfortunately, because of some personal reasons, I have to drop the project!  
I found [this](http://witness.report/) website, they're doing a pretty good job and I don't think I'm able to add more in a foreseeable future. Sorry.
